INSTALL

```diff
@@ -8,6 +8,7 @@
 
 To build and install the Squid Cache, type:
 
+        % ./bootstrap.sh
```

Comment:
INSTALL instructions are not for developers. Non-developers should not run bootstrap.sh -- it is unnecessary, likely to fail in their environments, complicates support, and failures may be difficult to recover from because they damage files extracted from the official tarball.
```suggestion
```